TROUBLESHOOTING.md

```diff
@@ -1,19 +1,20 @@
 # Troubleshooting Guide
 
-## Issues starting MySQL or PostgreSQL with docker-compose
+## Issues starting MySQL, PostgreSQL, or redis server with docker-compose
 
-If you have any trouble starting the MySQL/PostgreSQL database, make sure any database service is not already
-running outside of docker. On OSX, you can stop the MySQL and PostgreSQL services by executing:
+If you have any trouble starting the MySQL/PostgreSQL database or the redis server, make sure any database service is not already
+running outside of docker. On OSX, you can stop the MySQL, PostgreSQL, and redis server services by executing:
 ```bash
 brew services stop mysql
 brew services stop postgresql
+brew services stop redis-server
 ```
 or on Ubuntu with:
 ```bash
 service mysql stop
 service postgresql stop
+service redis-server stop
 ```
-
```

Comment:
nit: Do not delete this line. We want to leave one space before each new heading. (Also in general, please do not submit style changes that are not relevant to the PR).